code/modules/reagents/reagents/food.dm

```diff
@@ -76,6 +76,51 @@
 	counterlist_normalise(taste_amounts)
 
 	data = taste_amounts
+//various flavors of nutriment for protein bars
+/datum/reagent/consumable/protein
+	name = "Protein Mix"
+	description = "All the vitamins, minerals, and carbohydrates the body with a unique flavor"
+	nutriment_factor = 15
+	color = "#664330" // rgb: 102, 67, 48
+	taste_description = "nutraloafed food"
+	var/brute_heal = 1
+	var/burn_heal = 0
+	var/blood_gain = 0.4
+
+/datum/reagent/consumable/protein/on_mob_life(mob/living/L, metabolism)
```

Comment:
This looks really pretty useful. Is this copied from another food reagent?

Reply:
yeah its just nutriment, flavor is broken and i can't tell why from the code so this is a band-aid fix

Reply:
Fair enough